docs/source/install/docker-compose.rst

```diff
@@ -0,0 +1,275 @@
+
+.. warning::
+ these deployment are not intended for production given that 1. docker compose works on a single host and 2. permanent storage is left to docker to manage.
+
+.. note::
+    To learn more about docker compose, please visit https://docs.docker.com/compose/.
+
+Prerequisites
+*************
+
+It is required to install the following tools.
+
+    * **docker** Can be downloaded installed as explained in https://docs.docker.com/get-docker/
+
+    * **docker compose** Can be downloaded and installed as explained in https://docs.docker.com/compose/install/
+
+.. note::
+    At the time of writing this, all registry components were successfully tested with the docker compose version v2.2.3.
+    The registry components should work with later versions of docker compose too.
+
+Download and Configure the PDS Registry Application
+****************************************************
+
+The latest release of the Registry can be downloaded and configured as follows.
+
+1. Download the sourcecode `.zip` file or `.tar.gz` file from the https://github.com/NASA-PDS/registry/releases.
+
+2. Extract the **tar** (Linux, Mac) or **zip** (Windows) to a directory without spaces, such as */home/pds*.
+On Linux you can use the following command::
+
+    % tar -xzvf registry-x.y.z.tar.gz
+
+.. Note::
+    The path of the extracted directory, is referred to as the `<REGISTRY_ROOT>` in the following instructions.
+
+3. Open a terminal and change the working directory to `<REGISTRY_ROOT>/docker` directory.
+
+4. Open the `<REGISTRY_ROOT>/docker/.env` file using a text editor
+
+5. Edit the `.env` file and set the following environment variables. ::
+
+    # Absolute path of the Harvest data directory in the host machine (E.g.: `./test-data/registry-harvest-data`).
+    # If the Registry Harvest CLI is executed with the option to download test data, then this directory will be
+    # cleaned-up and populated with test data.
+    # Also, this `HARVEST_DATA_DIR` location should be accessible from the docker containers of the Registry Harvest Service
+    # and Registry Crawler Service.
+    HARVEST_DATA_DIR=./test-data/registry-harvest-data
+
+.. note::
+    The `HARVEST_DATA_DIR` specified above should have the directories of the data to be harvested.
+
+    An example of a `harvest-job-config.xml` file (to set as `HARVEST_JOB_CONFIG_FILE` in the above configurations)
+    can be obtained from the `<REGISTRY_ROOT>/docker/default-config` directory.
+
+5. Generate the self-signed certificates as follows.
+
+
+.. note::
+    Certificates are required for OpenSearch to enable https.
+..
+
+    a) Change the current working directory to `<REGISTRY_ROOT>/docker/certs` directory. ::
+
+        cd certs
+
+    b) Generate self-signed certificates by executing the following script. ::
+
+        ./generate-certs.sh
+
+    c) Change the current working directory to `<REGISTRY_ROOT>/docker` directory. ::
+
+        cd ..
+
+Deployment profiles
```

Comment:
Suggestion: It is better to start the work profiles with upper case P (Profiles) in this heading to be consistent with the [Title Case ](https://en.wikipedia.org/wiki/Title_case)used on most of the heading in the same heading level in this page (E.g.: Clean-up the Deployment, Download and Configure the PDS Registry Application)

Reply:
Done